src/js/components/Header/Header.js

```diff
@@ -1,32 +1,41 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 import styles from './Header.less';
 import Button from '../Button';
 
-const Header = () => (
-  <div className={styles.headerWrapper}>
-    <div className={styles.logoName}>
-      <div className={styles.logo}>
-        Logo
+const Header = () => {
+  const handleLogout = () => {
+    localStorage.removeItem('token');
+  };
+  const loginLink = props => <Link to="/login" {...props} />;
+  return (
+    <div className={styles.headerWrapper}>
```

Comment:
you know what to do with this infinite duplications) i'm talking about 'styles....'

Reply:
I'm not exactly understand what do you mean about styles duplications..)

Reply:
done)

Reply:
ty)